OracleGoldenGate/README.md

```diff
@@ -0,0 +1,236 @@
+
+   $ BASE_IMAGE="oracle/database:12.2.0.1-ee" ./dockerBuild.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip --build-arg BASE_COMMAND="runuser -u oracle -- /opt/oracle/runOracle.sh"
+
+### Running Oracle GoldenGate in a Docker container
+To run your Oracle GoldenGate Docker image use a **docker run** command like this:
+
+    $ docker run --name <container name> \
+        -e OGG_SCHEMA=<schema for OGG> \
+        -e OGG_ADMIN=<admin user name> \
+        -e OGG_ADMIN_PWD=<admin password> \
+        -e OGG_DEPLOYMENT=<deployment name for Microservices Architecture> \
+        -v <host mount point>:<container-mount-point> ... \
+        <image name>
+
+Parameters:
+
+- `<container name>`  - The name of the container (default: auto generated)
+- `-e OGG_SCHEMA`     - The GGSCHEMA to use for OGG (default: `oggadmin`)
+- `-e OGG_ADMIN`      - The name of the administrative account to create for Microservices Architecture (default: `oggadmin`)
+- `-e OGG_ADMIN_PWD`  - The password for the Microservices Architecture administrative account (default: auto generated)
+- `-e OGG_DEPLOYMENT` - The name of the deployment for Microservices Architecture (default: `Local`)
+- `<image name>`      - The Docker image name created using **Option 1** or **Option 2**
+
+**NOTE:** Only the `OGG_SCHEMA` environment variable is used by Oracle GoldenGate Standard Edition containers. The other environment variables are used by the Microservices Architecture.
+
+Mount points for Oracle GoldenGate Standard Edition are located in the container under the `/u01/app/ogg/` directory. For example:
+
+- `/u01/app/ogg/dirprm` - The parameter file directory
+- `/u01/app/ogg/dirdat` - The standard trail file directory
+
+For the Microservices Architecture, Oracle GoldenGate data is located under the `/u02/ogg` directory. Some examples are:
+
+- `/u02/ogg/Local/etc/conf`     - Configuration files for the 'Local' deployment
+- `/u02/ogg/Local/var/lib/data` - Trail files for the 'Local' deployment
+
+#### SSL Certificate for Microservices Architecture
+When the Oracle GoldenGate Docker image is created for Microservices Architecture, a dummy SSL certificate is generated for the OGG Web UI. Your own SSL certificate can be used instead of the dummy certificate like this:
+
+    docker run --name ogg-test \
+        -e OGG_SCHEMA=ggadmin \
+        -e OGG_ADMIN=oggadmin \
+        -v /path/to/certificate.pem:/etc/nginx/ogg.pem \
+        oracle/goldengate-microservices:12.3.0.1.0
+
+The certificate file, `/path/to/certificate.pem`, needs to contain a **BEGIN PRIVATE KEY** section, followed by one or more **BEGIN CERTIFICATE** sections. For example:
```

Comment:
I would be more specific and state that you need a full chain certificate including the private key and all intermediate and root CA public keys.